### PR TITLE
xorg-server: new article

### PR DESCRIPTION
**Pull request by mcpcpc**

### CODE REVIEW

File: xorg/xorg-server.txt [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+________________________________________________________________________________
+
+Depending on your graphics card hardware, one of the following graphics card 
+packages should also be installed:
+
+* xf86-video-amdgpu
+* xf86-video-ati
+* xf86-video-intel
+* xf86-video-nouveau
+* xf86-video-vesa
```
Comment (dylanaraps) [inline xorg/xorg-server.txt:37]: Should be noted that intel GPU users don't need to install `xf86-video-intel`. The generic modesetting driver (built into Xorg) works really well.

File: xorg/xorg-server.txt [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+|                    |                                                         |
+|   sowm             |   An itsy bitsy floating window manager. [4]            |
+|                    |                                                         |
++--------------------+---------------------------------------------------------+
+
+Remote X Session Management
+________________________________________________________________________________
+
+What if you need to access your applications remotely?  The solution is to 
+utilize X servers "baked in" X11 forwarding solution.  For for more information,
```
Comment (dylanaraps) [inline xorg/xorg-server.txt:78]: `servers` -> `server's`

File: xorg/xorg-server.txt [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+|   sowm             |   An itsy bitsy floating window manager. [4]            |
+|                    |                                                         |
++--------------------+---------------------------------------------------------+
+
+Remote X Session Management
+________________________________________________________________________________
+
+What if you need to access your applications remotely?  The solution is to 
+utilize X servers "baked in" X11 forwarding solution.  For for more information, 
+checkout the @/xorg/x11-forwarding article.
```
Comment (dylanaraps) [inline xorg/xorg-server.txt:79]: `checkout` -> `check out`

File: xorg/xorg-server.txt (hunk 1 above)
Comment (mcpcpc) [reply xorg/xorg-server.txt:37]: added as an additional note (almost word-for-word lol).

**Pull request merged**